src/routes/routes.js

```diff
@@ -21,4 +22,9 @@ router.post('/api/v1/send-email', Validation.emailValidation, UserAuth.emailSend
 router.post('/api/v1/reset-password/:token', verifyToken, Validation.passwordValidation, UserAuth.resetPassword);
 router.post('/api/v1/login', Validation.loginValidation, UserAuth.login);
 
+router.post('/api/v1/:userName/follow', verifyToken, UserFollow.followUser);
```

Comment:
The endpoint should be: `/api/profiles/:username/follow`. The project README has a guide to the endpoints design, please refer to it

Reply:
Well, thank you. Let me go for it.